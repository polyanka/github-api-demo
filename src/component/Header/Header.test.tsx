import { shallow } from 'enzyme';
import * as React from 'react';

import { Header } from './index';

describe('<Header />', () => {
  it('renders correctly Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
