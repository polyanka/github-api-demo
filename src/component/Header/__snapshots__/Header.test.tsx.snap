// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly Header component 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Typography))
      noWrap={true}
      variant="h6"
    >
      MyGitHub
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Link))
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      style={
        Object {
          "color": "#fff",
          "paddingLeft": 16,
        }
      }
      to="/"
    >
      Users
    </WithStyles(ForwardRef(Link))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
